### PR TITLE
Import highcharts css directly

### PR DESCRIPTION
**Pull request by wpears**
## Changes

- Imports highcharts css directly (`highcharts/...`) rather than relatively (`../../node_modules/highcharts/...`). I found that the behavior between npm and yarn differs when using a relative import into something included in `node_modules`, so this was breaking the yarn build in cf.gov

## Testing

`./setup.sh`

Everything should still look and behave as expected.

### CODE REVIEW

**Comment by anselmbradford** [comment]
Looks like I'm not sure why this was changed originally https://github.com/cfpb/cfpb-chart-builder/commit/1a1706703acf28f5fb1fc6307764269d156d2902

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cfpb-chart-builder",
-  "version": "5.0.0",
+  "version": "5.0.1",
```
Comment (anselmbradford) [inline package.json]: Do we want to do the bumping in the same PR? Okay if we're sure we'll be releasing right after this is merged.

Comment (wpears) [reply package.json]: Reverted while I debug the failure in npm

**Comment by wpears** [comment]
Update:

I discovered that this was due to the flatness of module installs in yarn vs. the more nested nature in npm (so `node_modules/highcharts` exists when installed via yarn but not npm).

https://github.com/cfpb/cfgov-refresh/pull/4655 adds highcharts to cfgov directly which is a workaround that allows this change to be included without breaking npm or yarn installs in the interim.

**Comment by anselmbradford** [comment]
Looks like npm totally got rid of the "problematic" `event-stream` versions.

**Comment by anselmbradford** [comment]
Annnnd bummer, we may need to wait for https://github.com/AveVlad/gulp-connect/pull/261

**Pull request merged**